src/helpers/dimensions.ts

```diff
@@ -303,6 +303,19 @@ function getScatterDimensions(mdObject: VisualizationObject.IVisualizationObject
     ];
 }
 
+export function getGeoChartDimensions(buckets: VisualizationObject.IBucket[]): AFM.IDimension[] {
+    const afm: AFM.IAfm = convertBucketsToAFM(buckets);
+    const attributes = afm.attributes || [];
+    return [
+        {
+            itemIdentifiers: [MEASUREGROUP],
+        },
+        {
+            itemIdentifiers: attributes.map(a => a.localIdentifier),
```

Comment:
please add the type for `a`

Reply:
How about `const { attributes = [] }: AFM.IAfm =  convertBucketsToAFM(buckets);`